### PR TITLE
Charter clean-up after CEP 52.

### PR DESCRIPTION
**Pull request by Zimmi48**
In particular, this removes a bunch of now resolved questions.

### CODE REVIEW

File: charter.md [hunk 1]
```diff
@@ -100,10 +86,10 @@ Technically the Coq platform will be a git/github/gitlab repo containing:
...
 
-- Move the current Windows Installer from core Coq to the platform
-- Simplify the windows installer in core Coq to only install Coq (and possibly very few selected plugins)
-- Ensure that platform CI runs with Coq master and release branches daily
-- Create a binary package for Linux (Snap store)
-- Create a binary package for Mac (Apple store)
-- Create an opam repo for the platform (or do this based on the existing Coq repo)
-- Provide developer setups on all platforms
+- [x] Move the current Windows Installer from core Coq to the platform
+- ~~Simplify the windows installer in core Coq to only install Coq (and possibly very few selected plugins)~~
```
Comment (MSoegtropIMC) [inline charter.md:90]: What does the double tilde mean here?

Maybe my wording was unclear. What I wanted to say is "decouple core Coq CI from all the packages the legacy Windows installer did build". This is implemented.

**Review by MSoegtropIMC — APPROVED** [review]
Except for my editorial comment LGTM.

**Comment by MSoegtropIMC** [comment]
@Zimmi48 : should we merge this to 2021.02?

File: charter.md (hunk 1 above)
Comment (Zimmi48) [reply charter.md:90]: The double tilde is strike-through. Indeed, this was not clear. I'll change the bullet point to your clearer statement and mark it as done.

**Comment by Zimmi48** [comment]
> @Zimmi48 : should we merge this to 2021.02?

Possibly, but primarily in `master` because when users discover the platform repo, the charter is advertised at the top of the README, so it's likely that they will read the `master` version.

**Comment by Zimmi48** [comment]
I targeted this PR at master because it was the default branch, but if that changes, then we should change the target.

**Comment by MSoegtropIMC** [comment]
@Zimmi48 : as discussed I will change the default branch later today to 2021.02. Can you please retarget this PR?

**Comment by MSoegtropIMC** [comment]
@Zimmi48 : it doesn't work that way - I guess you have to cherry pick or merge it over manually from a master based view to a 2021.02 based view. Shall I do it?

**Comment by Zimmi48** [comment]
Yes, please. I can also do it but not right now.

**Comment by MSoegtropIMC** [comment]
OK, I take care of it.

**Comment by MSoegtropIMC** [comment]
Done.

**Comment by Zimmi48** [comment]
Thanks!

**Pull request merged**